apps/src/dance/types.ts

```diff
@@ -46,3 +46,13 @@ export type AiModalReturnedItem = {
   name: string;
   url: string;
 };
+
+type EmojiAssociation = {
+  association: number[];
+  explanation: string;
+};
+
+export type CachedMapping = {
+  emojiAssociations: {[key: string]: EmojiAssociation};
+  output: string[];
+};
```

Comment:
If these are only used in a single file, I'd actually recommend putting them in that file, and just using this file for types that are used in more than one source file.

Reply:
Gotcha, migrated these over to the DanceAiClient file instead